### ISSUE TITLE
Component self-audit

### ISSUE THREAD

**Issue by trentmwillis** [trentmwillis/ember-axe#1]
Components ~~(and Views)~~ should be able to self-audit during development. This will allow developers to pick up on issues during development and actively maintain accessibility, rather than trying to fix it as an after thought during testing.

Aspects of this to implement (subject to change):
- [x] ~~Views~~ Components should automatically run the axe audit (scoped to their element) on any render event
- [x] Any extensions to ~~views~~ components should not ship in production code
- [x] ~~Views~~ Components should have easy-to-use hooks for the following:
  - [x] Turning off the tests
  - [x] Passing in configuration options
  - [x] Defining a custom callback to handle the results of the test
- [x] Audits should not run in the testing environment

**Note:** support for using views is not going to be implemented. Since Ember 1.13 and 2.0 are upon us, we're going to leverage the new Component life-cycle hooks (primarily, `didRender`) and look to the future.

**Comment by dylanb** [comment]
@trentmwillis you might want to consider a mode/option where the audits show up in the dev console in the way that Reast.js and Angular.js do with their audits.

**Comment by trentmwillis** [comment]
In the coming release, it logs any issues to the console with all the information obtained from the result payload. I'll probably try to format these into a more readable format thought.

I'm aware of the way react-a11y does it, is there a similar feature for Angular?

**Comment by trentmwillis** [comment]
Added feature in [version 0.0.2](https://github.com/trentmwillis/ember-axe/releases/tag/v0.0.2).

**Comment by dylanb** [comment]
Check out this http://marcysutton.github.io/angular-a11y/#/13 I don't know enough about it to know whether it comes standard or if it requires an extra module.
